src/main/webapp/resources/js/components/Buttons/LinkButton.jsx

```diff
@@ -20,6 +20,7 @@ export function LinkButton({ text, href, onClick, ...props }) {
         height: "auto",
         whiteSpace: "normal",
         textAlign: "left",
+        margin: 0,
```

Comment:
Can you make this component take a style from (add `style={}`)  to the arguments.  Then spread any styles onto the component styles.  This will allow the current styles to be overridden if needed.

Reply:
Please see 716fbdf82337db2076e265447692537ab6bfde23.